pkg/doc/verifiable/example_presentation_test.go

```diff
@@ -224,15 +225,125 @@ func ExamplePresentation_SetCredentials() {
 		},
 	}
 
+	vcStr := `
+{
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://transmute-industries.github.io/vc-greeting-card/context/vc-greeting-card-v0.0.jsonld"
```

Comment:
How does this external context affect tests?

Reply:
In this particular case, the context is not parsed, so it's OK. But I will change it to neutral `https://www.w3.org/2018/credentials/examples/v1`